pom.xml

```diff
@@ -216,6 +222,12 @@
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
```

Comment:
Yes, it's really strange, without the explicit dependency, a ClassNotFoundException is thrown